src/main/java/org/ssu/edu/teachua/ui/components/search/AdvancedSearchClubComponent.java

```diff
@@ -10,41 +10,75 @@
 
 public class AdvancedSearchClubComponent extends AdvancedSearchCenterComponent {
 
```

Comment:
you can add comments for class & package too

Reply:
added